phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerPanel.java

```diff
@@ -37,19 +38,24 @@
     public WinePrefixContainerPanel(WinePrefixContainerDTO containerEntity,
             List<EngineVersionDTO> engineVersions,
             EngineToolsManager engineToolsManager,
-            ApplicationDTO engineTools,
+            Optional<ApplicationDTO> engineTools,
             WinePrefixContainerController winePrefixContainerController) {
         super(containerEntity, engineVersions);
 
         this.informationTab = new WinePrefixContainerInformationTab(containerEntity, engineVersions);
+        this.tabPane.getTabs().add(this.informationTab);
         this.displayTab = new WinePrefixContainerDisplayTab(containerEntity, winePrefixContainerController);
+        this.tabPane.getTabs().add(this.displayTab);
         this.inputTab = new WinePrefixContainerInputTab(containerEntity);
-        this.wineToolsTab = new WinePrefixContainerWineToolsTab(containerEntity, winePrefixContainerController,
-                engineToolsManager, engineTools);
+        this.tabPane.getTabs().add(this.inputTab);
+        if (engineTools.isPresent()) {
```

Comment:
Maybe we can add a panel with a message like "no engine tools available" in case no repository is found?

Reply:
I don't know if this makes sense because as a user, there's nothing you could do about it. So what are you supposed to do if you see that message?

Reply:
I've misread. I thought you always add the tab.